app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -111,19 +111,34 @@ def update_classification
       form526_submission_id: id
     }
 
+    Rails.logger.info(
+      "classifying 526Submission... id: #{id}, saved_claim_id: #{saved_claim_id}"
+    )
     classification = classify_by_diagnostic_code(params)
-    update_form_with_classification(classification['classification_code']) if classification.present?
+    update_form_with_classification(classification)
   end
 
   # check claims
   def classify_by_diagnostic_code(params)
     vro_client = VirtualRegionalOffice::Client.new
     response = vro_client.classify_contention_by_diagnostic_code(params)
-    response.body
+    response = response.body
+
+    if response['responseBody'] # workaround so works w/ multiple versions of the VRO code
+      response = response['responseBody']
+    end
+    response
   end
 
-  def update_form_with_classification(_classification_code)
-    # TODO: update form[FORM_526] to include the classification code
+  def update_form_with_classification(classification)
+    classification_code = nil
+    classification_code = classification['classification_code'] if classification.present?
+    form[Form526Submission::FORM_526]['form526']['disabilities'].each do |disability|
+      disability['classificationCode'] = classification_code
+    end
+
+    update!(form_json: JSON.dump(form))
```

Comment:
I think JSON.dump does the same thing but I usually see people use to_json over JSON.dump.... not entirely sure it matters though.  

`form.to_json` instead of `JSON.dump(form)`